jsface.js

```diff
@@ -174,6 +195,15 @@
       clazz[key] = statics[key];
     }
 
+    // copy immutable properties from consts to clazz and freeze them recursively
```

Comment:
Extract this logic to $const plugin under Class.plugins. See: https://github.com/tnhu/jsface/blob/master/jsface.js#L190 as a reference.